pkg/syncer/metadatasyncer.go

```diff
@@ -93,6 +94,15 @@ var (
 	nodeMgr node.Manager
 	// isPodVMOnStretchSupervisorFSSEnabled is true when PodVMOnStretchedSupervisor FSS is enabled.
 	isPodVMOnStretchSupervisorFSSEnabled bool
+
+	// ResourceAPIgroupPVC represents resource API group for PVC resource.
+	ResourceAPIgroupPVC = ""
```

Comment:
why this is empty string?

Reply:
The crd doc says "For resources not in the core API group, this field is required". In this case, the resource kind is PVC.

Reply:
PVC is a core API resource. The resource group for PVC is "".  We can clarify in the comment.

Reply:
Done